pages/dkp/kommander/2.1/backup-and-restore/index.md

```diff
@@ -197,6 +197,46 @@ To restore cluster data on demand from a selected backup snapshot available in t
 velero restore create --from-backup BACKUP-NAME
 ```
 
+#### Restoring Kommander Management Cluster
+
+<p class="message--note"><strong>NOTE: </strong>When restoring a backup to the Kommander Management Cluster, you must adjust configuration to avoid restore errors.
+</p>
+
+Before running a restore make sure that the following `ResourceQuota` configuration is not configured on your cluster:
+
+```sh
+kubectl -n kommander delete resourcequota one-kommandercluster-per-kommander-workspace
+```
+
+When the restore is completed, the manually removed `ResourceQuota` named `one-kommandercluster-per-kommander-workspace` will automatically be restored.
+
+You must also turn off the `Workspace` validation webhooks which would prevent restoring Workspaces with pre-configured namespaces. If the validation webhook named `kommander-validating` is present it must be modified with the command:
```

Comment:
Since I don't know much in the topic, I see two different interpretations, so here are my suggestions: 
```suggestion
Turn off the `Workspace` validation webhooks. Otherwise, you will not restore Workspaces with pre-configured namespaces. If the validation webhook named `kommander-validating` is present, it must be modified with the command:
```
```suggestion
Turn off the `Workspace` validation webhooks to prevent restoring Workspaces with pre-configured namespaces. If the validation webhook named `kommander-validating` is present, it must be modified with the command:
```